docs/en/reference/information_schema/pipe_files.md

```diff
@@ -0,0 +1,24 @@
+---
+displayed_sidebar: "English"
+---
+
+# pipe_files
+
+`pipe_files` provides the status of the data files to be loaded via a specified pipe. This view is supported from StarRocks v3.2 onwards.
+
+The following fields are provided in `pipe_files`:
+
+| **Field**        | **Description**                                              |
+| ---------------- | ------------------------------------------------------------ |
+| DATABASE_NAME    | The name of the database in which the pipe is stored.        |
+| PIPE_ID          | The unique ID of the pipe.                                   |
+| PIPE_NAME        | The name of the pipe.                                        |
+| FILE_NAME        | The number of the data file.                                 |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/fb0add28-d581-44f1-96cc-6dd7f03481b7)
-> name
(This one should blame on me.)